tx-pool/src/process.rs

```diff
@@ -671,6 +694,23 @@ impl TxPoolService {
         }
     }
 
+    // # Notice
+    //
+    // This method assumes that the inputs transactions are sorted.
+    async fn try_process_txs(&self, txs: Vec<TransactionView>) {
+        if txs.is_empty() {
+            return;
+        }
+        let total = txs.len();
+        let mut count = 0usize;
+        for tx in txs {
+            if self._process_tx(tx, None).await.is_err() {
+                count += 1;
+            }
+        }
+        info!("{}/{} transactions are failed to process", count, total);
```

Comment:
Log enhancement suggestions:

- Log the error when `self._process_tx` fails, which may help users diagnose the problem.
- Log `info!("{}/{} transactions are failed to process", count, total);` only when `count > 0`. Because I think "failed" is a logging keyword that means there is something wrong.